Makefile

```diff
@@ -59,11 +61,11 @@ rpm: $(PACKETNAME).tar.gz $(PACKAGE_NAME).spec
 
 deb:
 	debuild -i -us -uc -b
-	make distclean
+	rm -rf build
```

Comment:
I don't think this is necessary.
`distclean` was there only because deb build
uses different prefix, so `make install` after that
will hurt. With your changes we likely need neither
`distclean`, nor `rm -rf`.